tutorials/nlp/GLUE_Benchmark_tutorial.ipynb

```diff
@@ -0,0 +1,540 @@
+        "colab_type": "text"
+      },
+      "source": [
+        "## Building the PyTorch Lightning Trainer\n",
+        "\n",
+        "NeMo models are primarily PyTorch Lightning modules - and therefore are entirely compatible with the PyTorch Lightning ecosystem.\n",
+        "\n",
+        "Let's first instantiate a Trainer object"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "1tG4FzZ4Ui60",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "print(\"Trainer config - \\n\")\n",
+        "print(OmegaConf.to_yaml(config.trainer))"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "knF6QeQQdMrH",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "# lets modify some trainer configs\n",
+        "# checks if we have GPU available and uses it\n",
+        "cuda = 1 if torch.cuda.is_available() else 0\n",
+        "config.trainer.gpus = cuda\n",
+        "\n",
+        "config.trainer.precision = 16 if torch.cuda.is_available() else 32\n",
+        "\n",
+        "# for mixed precision training, uncomment the line below (precision should be set to 16 and amp_level to O1):\n",
+        "# config.trainer.amp_level = O1\n",
+        "\n",
+        "# remove distributed training flags\n",
+        "config.trainer.distributed_backend = None\n",
+        "\n",
+        "# setup max number of steps to reduce training time for demonstration purposes of this tutorial\n",
+        "config.trainer.max_steps = 5\n",
+        "\n",
+        "trainer = pl.Trainer(**config.trainer)"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "8IlEMdVxdr6p",
+        "colab_type": "text"
+      },
+      "source": [
+        "## Setting up a NeMo Experiment¶\n",
+        "\n",
+        "NeMo has an experiment manager that handles logging and checkpointing for us, so let's use it:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "8uztqGAmdrYt",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "config.exp_manager.exp_dir = WORK_DIR\n",
+        "exp_dir = exp_manager(trainer, config.get(\"exp_manager\", None))\n",
+        "\n",
+        "# the exp_dir provides a path to the current experiment for easy access\n",
+        "exp_dir = str(exp_dir)\n",
+        "exp_dir"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "8tjLhUvL_o7_",
+        "colab_type": "text"
+      },
+      "source": [
+        "Before initializing the model, we might want to modify some of the model configs. For example, we might want to modify the pretrained BERT model, for example use Megatron-LM BERT or AlBERT model:"
```

Comment:
A"L"BERT. Please add links to Albert and megatron

Reply:
done